rustworkx-core/src/centrality.rs

```diff
@@ -1133,11 +1133,26 @@ where
         let index = graph.to_index(node);
         node_indices[index] = Some(node);
     });
+
+    let unweighted_shortest_path = |g: Reversed<&G>, s: G::NodeId| -> HashMap<G::NodeId, usize> {
+        let mut distances = HashMap::new();
+        let mut bfs = Bfs::new(g, s);
+        distances.insert(s, 0);
+        while let Some(node) = bfs.next(g) {
+            let distance = distances[&node];
+            for edge in g.edges(node) {
+                let target = edge.target();
```

Comment:
I was thinking we could do `for target in g.neighbors_directed(node, Outgoing)` here, but that would require we add the trait bound `IntoNeighborsDirected` on `G` and that seems unnecessary so this is fine.